sfge/src/main/java/com/salesforce/config/UserFacingMessages.java

```diff
@@ -12,6 +12,7 @@ public static final class RuleDescriptions {
                 "Identfies Apex operations that dereference null objects and throw NullPointerExceptions.";
         public static final String UNIMPLEMENTED_TYPE_RULE =
                 "Identifies abstract classes and interfaces that are non-global and don't have implementations or extensions.";
+        public static final String UNUSED_METHOD_RULE = "Identifies methods that are not invoked";
```

Comment:
@teresa-allen-sfdc , this is the existing description. Now that the rule is path-based, do we want to change the description to mention paths?

Reply:
@jfeingold35 I started a UI text doc to collaborate in.
https://docs.google.com/document/d/1oMhNKy1RbwLBs_ScvIRp8m6jlNT9WPlvWi1ew17U_JM/edit?usp=sharing